### PR TITLE
Standarize import names, fix typo, disable exhaustivestruct linter

### PR DESCRIPTION
**Pull request by invidian**
See commit messages for details.

### CODE REVIEW

File: pkg/agent/agent.go [hunk 1]
```diff
@@ -9,12 +9,12 @@ import (
 	"github.com/coreos/go-systemd/login1"
```
Comment (knrt10) [inline pkg/agent/agent.go:9]: Can you please change the following URLs in the commit message to the following? It does not look nice as it exceeds 72 lines. I have already changed then to short URL. You can verify and change it

- shorturl.at/eijzA
- shorturl.at/eqATY

**Review by knrt10 — COMMENTED** [review]
Small NIT, rest LGTM

File: pkg/agent/agent.go (hunk 1 above)
Comment (invidian) [reply pkg/agent/agent.go:9]: Hm, maybe I Base 64 them instead? :smile:

Comment (knrt10) [reply pkg/agent/agent.go:9]: if it short then, then yes 😄

Comment (invidian) [reply pkg/agent/agent.go:9]: Sort of, then line breaks can be added. Jokes aside, https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53#gistcomment-2355038 suggests adding long URLs via reference. What do you think?

Comment (knrt10) [reply pkg/agent/agent.go:9]: Cool, that's fine then

Comment (invidian) [reply pkg/agent/agent.go:9]: Pushed with references.

**Review by knrt10 — APPROVED** [review]

**Pull request merged**